src/App.vue

```diff
@@ -56,7 +60,8 @@ export default {
       return ~~(this.length * this.percentage) + 1
     },
     bandStyle () {
-      const x = -1040 * this.current
+      const width = (this.isMounted && this.$refs.main.offsetWidth < 1040) ? this.$refs.main.offsetWidth : 0
+      const x = width * -1 * this.current
```

Comment:
It seems that `x` will always be 0 when screen width is greater than 1040, so it won't work correctly on PCs.